CorsixTH/Lua/languages/brazilian_portuguese.lua

```diff
@@ -872,29 +949,10 @@ map_editor_window = {
     heliport_3 = "Heliporto 3",
     heliport_4 = "Heliporto 4",
     paste = "Colar área",
-  }
-}
-
-fax = {
-vip_visit_result = {
-  remarks = {
-    [1] = "Que hospital eficaz! A próxima vez que estiver gravemente doente, eu venho aqui.",
-    [2] = "É isto o que eu chamo de um hospital.",
-    [3] = "É um hospital excelente. E sei o que digo; visito muitos hospitais.",
-    [4] = "Que hospital bem administrado! Obrigado pelo seu convite.",
-    [5] = "Mmmm. Certamente näo é uma má instituiçäo médica.",
-    [6] = "Gostei do seu estupendo hospital. Agora, alguém quer um curry da Casa do Curry?",
-    [7] = "Bom, já vi piores. Mas deveria fazer algumas melhoras.",
-    [8] = "Oh céus! Näo é um lugar agradável para ir se estiver doente.",
-    [9] = "Vou ser sincero, é um hospital normal. Francamente, eu esperava mais.",
-    [10] = "Por que me zanguei? Foi pior que ir ver uma ópera de quatro horas!",
-    [11] = "Estou aborrecido pelo que vi. Chama a isto um hospital? Mais parece uma pocilga!",
-    [12] = "Estou farto de atrair a atençäo pública e de visitar buracos fedorentos como este! Eu me demito.",
-    [13] = "Que lixo. Vou tentar fechá-lo.",
-    [14] = "Eu nunca vi um hospital täo ruim. Que vergonha!",
-    [15] = "Estou chocado. Você näo pode chamar isso de hospital! Näo me espere ver novamente.",
-    }
-  }
```

Comment:
Did you mean to delete these? We still use these and I can't see it elsewhere in this file.

Reply:
Oh, thank you